src/analytics/mod.rs

```diff
@@ -45,39 +47,37 @@ impl PushAnalytics {
     ) -> Result<Self> {
         info!(%export_bucket, "initializing analytics with aws export");
 
-        let opts = BatchCollectorOpts::default();
+        let opts = BatchOpts::default();
         let bucket_name: Arc<str> = export_bucket.into();
         let node_ip: Arc<str> = node_ip.to_string().into();
 
         let messages = {
-            let exporter = AwsExporter::new(AwsExporterOpts {
+            let exporter = AwsExporter::new(AwsOpts {
+                // TODO (Harry Bairstow): Do we need a prefix?
+                export_prefix: "",
                 export_name: "push_messages",
                 file_extension: "parquet",
-                // Note: Clone these values if we add more exporters
                 bucket_name: bucket_name.clone(),
                 s3_client: s3_client.clone(),
                 node_ip: node_ip.clone(),
             });
 
-            Analytics::new(
-                gorgon::batcher::create_parquet_collector::<MessageInfo, _>(opts.clone(), exporter)
-                    .map_err(|e| Error::BatchCollector(e.to_string()))?,
-            )
+            let collector = ParquetWriter::<MessageInfo>::new(opts.clone(), exporter)?;
+            Analytics::new(collector)
         };
 
         let clients = {
-            let exporter = AwsExporter::new(AwsExporterOpts {
+            let exporter = AwsExporter::new(AwsOpts {
+                // TODO (Harry Bairstow): Do we need a prefix?
+                export_prefix: "",
```

Comment:
did we have one before?

Reply:
Not for Echo Server, but `gorgon` requires this now and the relay has one, going to investigate with Data team